### PR TITLE
Moving initdb functions into InitDB class

### LINKED ISSUE #4225: Create an InitDB class
i think it would be cleaner to put the functions in pootle.core.initdb into a class

### PR DESCRIPTION
**Pull request by phlax**
Fixes #4225

### CODE REVIEW

File: pootle/apps/pootle_app/management/commands/initdb.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class Command(NoArgsCommand):
 
     def handle_noargs(self, **options):
         self.stdout.write('Populating the database.')
-        initdb(options["create_projects"])
+        InitDB().init(options["create_projects"])
```
Comment (julen) [inline pootle/apps/pootle_app/management/commands/initdb.py]: Maybe the naming is misleading and the intention is different, so correct me if I'm wrong, but I find this construct totally counter-intuitive — there's `__init__` for a purpose.

Comment (phlax) [inline pootle/apps/pootle_app/management/commands/initdb.py]: fair point - ill change

**Comment by unho** [comment]
I have no additional concerns. lgtm once the raised concern is fixed.

File: pootle/apps/pootle_app/management/commands/initdb.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_app/management/commands/initdb.py]: I thought that this would be changed to pass the options to `__init__` instead of `initdb`.

Comment (phlax) [inline pootle/apps/pootle_app/management/commands/initdb.py]: no - i just changed the name of the method - i didnt want to trigger the db install on object creation for several reasons

Comment (unho) [inline pootle/apps/pootle_app/management/commands/initdb.py]: I was talking about using `InitDB(options["create_projects"]).init_db()` instead.

Comment (phlax) [inline pootle/apps/pootle_app/management/commands/initdb.py]: seem more like an option to the init_db method than the constructor to me

**Comment by julen** [comment]
Man, mind your commit messages :pray:

**Pull request merged**